Kudu.Services.Web/WebHooks.cshtml

```diff
@@ -50,7 +50,15 @@
     </table>
 </div>
 
+<script>
+    var knockoutJsLoaded = false;
+</script>
 <script src="//ajax.aspnetcdn.com/ajax/knockout/knockout-2.2.1.js"></script>
```

Comment:
do you need to set `knockoutJsLoaded` here?

Reply:
Yes. thanks for noticing this. As we are planning to use local dependency I will just change the src = <local path>

---
In reply to: [251172743](https://github.com/projectkudu/kudu/pull/2922#discussion_r251172743) [](ancestors = 251172743)